### PR TITLE
Improve emoji picker look

### PR DESCRIPTION
**Pull request by Gargron**
screenshot from 2017-04-11 01-59-37

### CODE REVIEW

File: app/assets/stylesheets/components.scss [hunk 1]
```diff
@@ -1575,11 +1597,15 @@ button.active i.fa-retweet {
...
   .emoji-search-wrapper + .emoji-categories-wrapper {
-    top: 83px;
+    top: 93px;
   }
 
   .emoji-row .emoji:hover {
     background: lighten($color2, 3%);
+
+    img, svg {
+      transform: translateZ(0) scale(1.2);
```
Comment (riking) [inline app/assets/stylesheets/components.scss]: the translateZ here is fishy - do you really need a separate render plane for EVERY image?

**Review by riking — CHANGES_REQUESTED** [review]
Just a perf nit in the CSS.

File: app/assets/stylesheets/components.scss (hunk 1 above)
Comment (Gargron) [inline app/assets/stylesheets/components.scss]: Hmm. I think I was misinformed about the performance implications - I believed that `translateZ(0)` activates GPU to make animations smoother (and I am planning to add a bouncy animation to this at some point.

**Comment by riking** [comment]
Well, you can add the translateZ when it's actually used, right?
And yeah it creates a separate GPU context for the element which makes
animations easier.

**Comment by Gargron** [comment]
@riking Added animation!

**Comment by krainboltgreene** [comment]
This is so hecking legit.

**Review by krainboltgreene — APPROVED** [review]

**Pull request merged**